Hotel-Module/Hotel/Controllers/LoginController.cs

```diff
@@ -25,20 +30,61 @@ public LoginController(ILogger<LoginController> logger, IHttpClientFactory httpC
             //var t = httpClient.GetAsync("endpoint");
         }
 
-        public IActionResult Index()
+        [HttpGet("/")]
+        public IActionResult Home()
         {
-            return View();
+            return Redirect("/profile");
         }
 
-        public IActionResult Privacy()
-        { 
-            return View();
+        [HttpGet("/login")]
+        public IActionResult Login()
+        {
+            return View("~/Views/Login/Index.cshtml");
+        }
+
+        [HttpPost("/login")]
+        public async Task<IActionResult> Login([FromForm] string authString)
+        {
+            if(authString is null)
+            {
+                return View("~/Views/Login/Index.cshtml");
+            }
+
+            try
+            {
+                HttpClient httpClient = httpClientFactory.CreateClient();
+                HttpRequestMessage httpRequest = new HttpRequestMessage();
+                httpRequest.Headers.Add(ServerApiConfig.TokenHeaderName, authString);
+                httpRequest.Method = HttpMethod.Get;
+                httpRequest.RequestUri = new Uri($"{ServerApiConfig.BaseUrl}/rooms?pageNumber=1&pageSize=1");
+                HttpResponseMessage httpResponse = await httpClient.SendAsync(httpRequest);
+                if (httpResponse.IsSuccessStatusCode)
```

Comment:
I think this could be simplified like below. This way `BaseUrl` doesn't need to be remembered in `ServerApiConfig` class as it is set only once at Startup in `httpClient.BaseAddress` property.

```suggestion
            try
            {
                HttpClient httpClient = httpClientFactory.CreateClient(nameof(DefaultHttpClient));
                httpClient.DefaultRequestHeaders.Add(ServerApiConfig.TokenHeaderName, authString);
                HttpResponseMessage httpResponse = await httpClient.GetAsync("rooms?pageNumber=1&pageSize=1");
                if (httpResponse.IsSuccessStatusCode)
```